client/src/test/java/com/paypal/selion/reports/excelreport/ExcelReportTest.java

```diff
@@ -50,7 +50,7 @@
  */
 public class ExcelReportTest {
 
-    private String excelReportFileName = "dummy_excel_test.xls";
+    private final String excelReportFileName = "dummy_excel_test.xls";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120392/issues/source?bid=2928425&fileBranchId=2943483#l53)